jflac-codec/src/main/java/org/jflac/FLACDecoder.java

```diff
@@ -748,6 +767,8 @@ public Frame readNextFrame() throws IOException {
                     readFrame();
                     return frame;
                 } catch (FrameDecodeException e) {
+                	//e.printStackTrace();
```

Comment:
Hi, sorry for the late reply. No, this was done by justflac, but I agree to this: It does not make sense to print a stack trace for every failed decoded frame, this is even dangerous for production (where does the print go to, can it digest huge amounts of data etc).